src/mb_enhanced_cover_art_uploads/providers/amazon.ts

```diff
@@ -73,7 +81,8 @@ export class AmazonProvider extends CoverArtProvider {
     #extractFrontCover(pageDom: Document): CoverArt | undefined {
         const frontCoverSelectors = [
             '#digitalMusicProductImage_feature_div > img', // Streaming/MP3 products
-            'img#main-image', // Audible products
+            'img#main-image', // Audible products, /dp URLs
```

Comment:
Hmm, probably it's possible after all to split this into two providers, if we simply remove this selector for Audible products on `dp` URLs (which would make the music provider fail for audiobooks). The selector should even be no longer in use as the redirect to the other Audible URL format happens before we reach this method (unless I have missed something).
```suggestion
```